df_py/util/vesting_schedule.py

```diff
@@ -26,8 +26,7 @@ def get_active_reward_amount_for_week_eth_by_stream(
     dfweek = get_df_week_number(start_dt) - 1
 
     if substream == "predictoor":
-        # 0.01%
-        return total_reward_amount * 0.001 if dfweek >= PREDICTOOR_RELEASE_WEEK else 0
+        return 35000 if dfweek >= PREDICTOOR_RELEASE_WEEK else 0
```

Comment:
I think this 35000 should be more visible, in the form of a constant or something similar. Also, can you please format it as `35_000` for better readability?

Reply:
Fully agree. We should avoid having magic numbers randomly scattered throughout the code. Rather, put them in https://github.com/oceanprotocol/df-py/blob/main/df_py/util/constants.py. 

Same  goes for other magic numbers.